docs/lessons/05-data-structures-lists.md

```diff
@@ -10,6 +10,99 @@
 
 <GoogleSlides src="https://docs.google.com/presentation/d/e/2PACX-1vSr6F6309N3wBgzfe9wGyP-vbW7q794IycXIBDTG0yUwONrLYB3Z3S0EoDnv3-B4D0vOqb8pFLU6Zgp/embed?start=false&loop=false&delayms=3000"></GoogleSlides>
 
+## Exercises
+
+### Exercise 1
+Build a train using ArrayList and LinkedList. Create an ArrayList that holds the waggons, then create a LinkedList that contains the waggons and has also a locomotive. Print the train in the console.
```

Comment:
I find it nice to wrap class names and values into "`" 

so that they are shown formatted like `ArrayList` etc.